script.js

```diff
@@ -0,0 +1,155 @@
+//DOM요소 가져오기
+const dateInfo = document.getElementById('date');
+const input = document.querySelector('input');
+const form = document.querySelector('form');
+const todoContent = document.querySelector('.todoContent');
+const doneContent = document.querySelector('.doneContent');
+const todoNum = document.getElementById('todoNum');
+const doneNum = document.getElementById('doneNum');
+
+let todoList = [];
+let doneList = [];
+
+//날짜 띄우는 함수
+const clock = () => {
+  let current = new Date(); //현재 날짜 와 시간
+  let year = current.getFullYear(); //연도
+  let month = current.getMonth(); //월
+  let date = current.getDate(); //일
+  let day = current.getDay(); //요일
+  var hours = current.getHours(); // 시간
+  const dayList = ['일', '월', '화', '수', '목', '금', '토'];
+
+  dateInfo.innerText = `${year}.${month + 1}.${date} ${
+    dayList[day]
+  } ${hours}시`;
+};
+
+//submit이벤트 핸들러
+form.addEventListener('submit', function (event) {
+  // 기본 제출 동작을 막고 할일 추가함수 실행
+  event.preventDefault();
+  inputTodo();
+});
+
+//할일을 입력받아 todoList에 추가 후 리렌더링
+const inputTodo = () => {
+  //입력값 유효성 검사
+  if (input.value.trim() === '') {
+    return;
+  }
+  //배열추가
+  todoList.push(input.value);
+  localStorage.setItem('todos', JSON.stringify(todoList));
+  input.value = '';
+
+  //리렌더링
+  renderTodo();
+};
+
+//로컬스토리지 내 todo 목록 띄우기
+const renderTodo = () => {
+  todoList = JSON.parse(localStorage.getItem('todos')) || [];
+  todoNum.innerText = todoList.length;
+  todoContent.innerHTML = ''; //html 초기화
+  todoList.forEach((todo, index) => {
+    //요소 생성 후 띄우기
+    const li = document.createElement('li');
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.addEventListener('change', () => {
+      addDone(todo);
+      removeTodo(index);
+    });
+
+    const span = document.createElement('span');
+    span.innerText = todo;
+
+    const deleteBtn = document.createElement('button');
+    deleteBtn.innerText = 'X';
+    deleteBtn.addEventListener('click', () => removeTodo(index));
+
+    // li 요소에 체크박스, 제목, 삭제 버튼을 추가
+    li.appendChild(checkbox);
+    li.appendChild(span);
+    li.appendChild(deleteBtn);
```

Comment:
단순히 todo text를 넣기 위핸 span을 선언하기보다
```suggestion
    const deleteBtn = document.createElement('button');
    deleteBtn.innerText = 'X';
    deleteBtn.addEventListener('click', () => removeTodo(index));

    // li 요소에 체크박스, 제목, 삭제 버튼을 추가
    li.appendChild(checkbox);
    li.appendChild('todo');
    li.appendChild(deleteBtn);
```
와 같이도 적어줄 수 있을 것 같습니다